src/index.js

```diff
@@ -0,0 +1,79 @@
+import renderInfo from './weather/temp';
+
+const inputEl = document.getElementById('city-name');
+const searchBtn = document.getElementById('search-btn');
+const metricBtn = document.querySelector('.metric');
+const imperialBtn = document.querySelector('.imperial');
+
+function filterWeatherData(data) {
+  return {
+    id: data.weather[0].id,
+    name: data.name,
+    description: data.weather[0].description,
+    icon: data.weather[0].icon,
+    temp: Math.round(data.main.temp),
+    feels: Math.round(data.main.feels_like),
+    humidity: data.main.humidity,
+    wind: data.wind.speed,
+    pressure: data.main.pressure,
+  };
+}
```

Comment:
Make use of arrow functions instead of the `function` keyword ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))